pwa/src/templates/templateParts/formHeader/FormHeaderTemplate.tsx

```diff
@@ -35,21 +36,25 @@ export const FormHeaderTemplate: React.FC<FormHeaderTemplateProps> = ({
         {customElements && customElements}
 
         {formId && (
-          <Button type="submit" form={formId} {...{ disabled }}>
+          <Button type="submit" form={formId} {...{ disabled }} className={styles.buttonContentWrap}>
             <FontAwesomeIcon icon={faFloppyDisk} />
             {t("Save")}
           </Button>
         )}
 
         {handleToggleDashboard && (
-          <Button {...{ disabled }} onClick={handleToggleDashboard.handleToggle}>
+          <Button {...{ disabled }} onClick={handleToggleDashboard.handleToggle} className={styles.buttonContentWrap}>
             <FontAwesomeIcon icon={handleToggleDashboard.isActive ? faMinus : faPlus} />
             {handleToggleDashboard.isActive ? t("Remove from dashboard") : t("Add to dashboard")}
           </Button>
         )}
 
         {handleDelete && (
-          <Button {...{ disabled }} onClick={handleDelete} className={styles.deleteButton}>
+          <Button
+            {...{ disabled }}
+            onClick={handleDelete}
+            className={clsx(styles.deleteButton, styles.buttonContentWrap)}
+          >
```

Comment:
Instead of adding these class names for every button, we can target them without adding the class names, like so:

```
.buttonsContainer > * {
  white-space: nowrap;
}
```